view/frontend/web/css/source/gutenberg-blocks/_block-media-text.less

```diff
@@ -0,0 +1,29 @@
+.wp-block-media-text {
+    display: grid;
+    grid-column-gap: @gap-size;
+    grid-template: auto / 50% auto;
```

Comment:
@orofabio have you tested this?
This might need escaping from the less compiler.